src/main/java/Event.java

```diff
@@ -0,0 +1,43 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    String at;
```

Comment:
Hello! Everything is good here. It'll be better to add a protected/private for String at and the one below it!